libs/ui/src/lib/form-message/form-message.component.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  HostBinding,
+} from '@angular/core';
+import { CviIconName } from '@egov/cvi-icons';
+import { FormMessageSeverity } from './form-message';
+import { FormMessageSeverityToHeaderIconPipe } from './form-message-severity-to-header-icon.pipe';
+
+@Component({
+  selector: 'cvi-ng-form-message',
+  templateUrl: './form-message.component.html',
+  providers: [FormMessageSeverityToHeaderIconPipe],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class FormMessageComponent {
+  message = '';
+  display!: boolean;
```

Comment:
This prop doesn't make much sense... Apparently it only is valuable to Storybook? If yes then should it be removed?

Or is it like this that without this prop the message box cannot be rendered because of how directives work? If that is the case maybe we can rename it to `displayMessage` since right now in template code it's a bit confusing what does the prop do.